apps/test-blinky/src/TutorialServo.cpp

```diff
@@ -0,0 +1,42 @@
+//
+//  TutorialServo.cpp
+//  
+//
+//  Created by Ari Wasch on 9/18/20.
+//
+
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2) : _pin(pin) {
+    _pin = 0;
```

Comment:
This constructor doesnt look right. You shouldnt need to store the pin. The pin is only used to init the pwm. Also, use the member initializer list to injt member vars